AudreyW_Lab_1.ipynb

```diff
@@ -0,0 +1,598 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "ax = sns.distplot(data_poisson3,\n",
+    "                  bins=30,\n",
+    "                  kde=True,\n",
+    "                  color='skyblue',\n",
+    "                  hist_kws={\"linewidth\": 15,'alpha':1})\n",
+    "ax.set(xlabel='Poisson Distribution', ylabel='Frequency')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Statistical question: What is the likelihood that I will be in (blank) location?"
```

Comment:
Also, I think you should define an explicit location in your statistical question